packages/transaction-log/src/components/common/transactionlog.datatable.component.tsx

```diff
@@ -153,43 +167,31 @@ const TransactionLogTable: React.FC<{
-                                transaction.status === 'Processing'
-                                  ? 'rgba(33, 150, 243, 0.2)' // info.light with 20% opacity
-                                  : transaction.status === 'Pending Async'
-                                  ? 'rgba(33, 150, 243, 0.2)' // info.light with 20% opacity
-                                  : transaction.status === 'Successful'
-                                  ? 'rgba(76, 175, 80, 0.2)' // success.light with 20% opacity
-                                  : transaction.status === 'Completed'
-                                  ? 'rgba(255, 193, 7, 0.2)' // warning.light with 20% opacity
-                                  : transaction.status === 'Completed with error(s)'
-                                  ? 'rgba(255, 193, 7, 0.2)' // warning.light with 20% opacity
-                                  : 'rgba(244, 67, 54, 0.2)', // error.light with 20% opacity
-                            },
+                                : 'rgba(244, 67, 54, 0.2)' // error.light with 20% opacity
+                          }
+                        }}
+                      >
+                        <LockIcon
+                          sx={{
+                            color:
+                              transaction.status === 'Processing'
+                                ? 'info.main'
+                                : transaction.status === 'Pending Async'
+                                ? 'info.main'
+                                : transaction.status === 'Successful'
+                                ? 'success.main'
+                                : transaction.status === 'Completed'
+                                ? 'warning.main'
+                                : transaction.status ===
+                                  'Completed with error(s)'
+                                ? 'warning.main'
+                                : 'error.main'
                           }}
-                        >
-                          <LockIcon
-                            sx={{
-                              color:
-                                transaction.status === 'Processing'
-                                  ? 'info.main'
-                                  : transaction.status === 'Pending Async'
-                                  ? 'info.main'
-                                  : transaction.status === 'Successful'
-                                  ? 'success.main'
-                                  : transaction.status === 'Completed'
-                                  ? 'warning.main'
-                                  : transaction.status === 'Completed with error(s)'
-                                  ? 'warning.main'
-                                  : 'error.main',
-                            }}
-                          />
-                        </IconButton>
+                        />
+                      </IconButton>
```

Comment:
Suggest to use text and not button based on this feedback https://app.clickup.com/t/86c0fd2v2

Reply:
Fixed.